openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-video-util.js

```diff
@@ -147,31 +148,65 @@ var VideoUtil = (function() {
 		const processor = _processor || __processTopToBottom;
 		return processor(area, rectNew, list);
 	}
-	function _arrange() {
-		const list = [];
-		$(VIDWIN_SEL).each(function() {
-			const v = $(this);
-			v.css(_getPos(list, v.width(), v.height()));
-			list.push(_getRect(v));
-		});
+	function _arrange_set(v, list) {
+		var w, h, f, r = false;
+
+		switch (self.arrangeMode) {
+			case 1:
+				w = 240;
+				h = 180;
+				break;
+			case 2:
+				w = 120;
+				h = 90;
+				f = __processEqualsBottomToTop;
+				break;
+
+			default:
+				const v2 = v.find('.video').get(0);
+				w = v2 && parseInt(v2.style.width, 10);
```

Comment:
Am I right `, 10` cab omitted here?

Reply:
It truncate 'px' to make an integer.

Reply:
full syntax of `parseInt` is `parseInt(str, radix)`, `radix is `10` by default

so `parseInt(v2.style.width, 10);` === `parseInt(v2.style.width);`

Reply:
Ahh... cab = can :-))) No, base is not guaranteed to be 10:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
yep :)
I was in one-handed-baby-sitting-mode sorry for typo :(

I believe there are no chances size in `px` will start with `0` or `0x` .... :))

Reply:
I will omit it if you so desire. ^_^
I just had bad experience with its behaviour in the past, even in non-leading zero cases. Especially on weird platforms like set-top boxes. And always specify it now. Better be safe than sorry. ^_^

Reply:
OK, let it be :)